blog/exporters.md

```diff
@@ -1,19 +1,22 @@
 ---
-title: How to use Security Advisor with Skydive
+title: How to use Exporter pipeline with Skydive
 layout: blog-post
 author: Kalman Meth
 date: 31/07/2019
 ---
 
+The Exporter filters the flow data obtained from Skydive, performs a data transformation, and saves the information to some target (e.g. an object store).
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+We use the term `pipeline` to describe the sequence of operations performed on the data: `classify`, `filter`, `transform`, `encode`, `store`.
+The combinations of filter, transformation, encoder, etc, allow the exporter to be exploited for different types of applications.
+In addition to the core Exporter modules, we have a Security Advisor application that is built using the Exporter components.
```

Comment:
Rather write:

```
We also have a Security Advisor exporter which feeds the Secutiy Advisor application with flow data captured using Skydive.
```

Reply:
The sentence was removed and replaced with
"We describe below the use of the Flow Exporter for the Security Advisor application."

Reply:
ok